hash/leapster.xml

```diff
@@ -395,32 +459,36 @@ Known games listed by part-no, (*) denotes undumped:
 		<publisher>LeapFrog</publisher>
 		<info name="serial" value="500-13298-A" />
 		<part name="cart" interface="leapster_cart">
+			<feature name="pcb" value="300-30033" />
+			<feature name="u1" value="U1" /> <!-- 24C16A -->
+			<feature name="u2" value="U2" /> <!-- EPOXY BLOB ROM -->
+			<feature name="u3" value="U3" /> <!-- Unpopulated slot -->
```

Comment:
See here's a great example - none of the `feature` elements provide value - you'd be better off just putting "U3 is an unpopulated N-pin socket" in the comment.